packages/google-pay-integration/src/gateways/google-pay-gateway.ts

```diff
@@ -0,0 +1,182 @@
+import { round } from 'lodash';
+
+import {
+    BillingAddressRequestBody,
+    guard,
+    InvalidArgumentError,
+    NotInitializedError,
+    NotInitializedErrorType,
+    PaymentIntegrationService,
+    PaymentMethod,
+} from '@bigcommerce/checkout-sdk/payment-integration-api';
+
+import isGooglePayCardNetworkKey from '../guards/is-google-pay-card-network-key';
+import {
+    GooglePayCardDataResponse,
+    GooglePayCardNetwork,
+    GooglePayCardParameters,
+    GooglePayGatewayParameters,
+    GooglePayInitializationData,
+    GooglePayMerchantInfo,
+    GooglePaySetExternalCheckoutData,
+    GooglePayTransactionInfo,
+} from '../types';
+
+export default class GooglePayGateway {
```

Comment:
[Brilliant](https://giphy.com/clips/parksandrec-parks-and-recreation-rec-peacock-tv-KmC8ofTnCUKq6H3AzT)